doc/src/conf.py

```diff
@@ -372,7 +372,13 @@
 \usepackage{letltxmacro}
 \LetLtxMacro\OldLaTeX\LaTeX
 \AtBeginDocument{\DeclareRobustCommand{\LaTeX}{\text{\OldLaTeX}}}
-'''
+\let\OldUnderscore\_
+\makeatletter
+\AtBeginDocument{\sbox\sphinxcontinuationbox{\spx@opt@verbatimcontinued}}
+\makeatother
+\protected\def\_{\OldUnderscore\discretionary{}{\sphinxafterbreak}{}}
+''',
+    'sphinxsetup': 'verbatimforcewraps',
```

Comment:
log file continues to contain the few related `Overfull \hbox` (https://github.com/sphinx-doc/sphinx/issues/12722) but the problem is actually solved in PDF.  But the vast majority of disappearing contents is caused by https://github.com/sphinx-doc/sphinx/issues/12723.  There is also the problem of the index, but it is solved by https://github.com/sympy/sympy/pull/26866.